### DISCUSSION TITLE
doesn't work in ios11 & ios10

### DISCUSSION THREAD

**Discussion by lilixiang5473** [iddan/react-native-canvas#246] [Q&A]
![1578538565508](https://user-images.githubusercontent.com/42955228/72033946-acaea580-32ce-11ea-84c3-afd82b6ceb2e.jpg)
Do you know what happened?

**Comment by iddan** [comment]
Can you check with the newly released: v0.1.37

**Comment by kimmy-wang** [comment]
I have the same issue in ios13.

version: v0.1.37

[repo](https://github.com/upcwangying/react-native-canvas-demo)
